service/orchestrator/orchestrator.go

```diff
@@ -58,6 +60,12 @@ type Service struct {
 	// metricConfigurations holds a double-map of metric configurations associated first by service ID and then metric ID
 	metricConfigurations map[string]map[string]*assessment.MetricConfiguration
 
+	// Currently only in-memory
+	Results map[string]*assessment.AssessmentResult
```

Comment:
A thought of mine: Now that we have Results stored in the orchestrator I'm thinking about removing them from the assessment in the future and also "redirect" the CLI command `ListAssessmentResults` to the orchestrator? Unless it should be possible to use the Assessment without orchestrator (service notion).
Either way, we should provide somehow `ListAssessmentResults` in the orchestrator as well, since it can have results from multiple assessment tools

Reply:
@oxisto @lebogg I´ve added a new discussion: #422